src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionTestResultViewController.swift

```diff
@@ -61,15 +61,15 @@ class ExposureSubmissionTestResultViewController: DynamicTableViewController, Sp
 		guard let result = testResult else { return }
 		switch result {
 		case .positive:
-			setButtonTitle(to: AppStrings.ExposureSubmissionResult.continueButton)
-			hideSecondaryButton()
+			navigationFooterItem?.primaryButtonTitle = AppStrings.ExposureSubmissionResult.continueButton
```

Comment:
general remark: the code looked cleaner before these changes, considering the diff.

Reply:
What do you mean with 'cleaner'? Navigation related items have always been managed using a navigation item. This PR just replaced all custom calls modifying navigation controls to access the navigationItem instead.